.github/workflows/bump_plugins.yml

```
@@ -0,0 +1,109 @@
+name: Update plugins
+on:
+  workflow_dispatch:
+
+jobs:
+  # ugly way to share the plugin list
+  plugin_list:
+    runs-on: ubuntu-latest
+    outputs:
+      plugins: ${{ steps.set-list.outputs.plugins }}
+    steps:
+    - name: Write the plugin list
+      run: echo '["foreman_ansible"]' > plugin_list.json
+    - name: Set the plugins list
+      id: set-list
+      run: |
+        PLUGINS=$(echo $(cat plugin_list.json))
+        echo "::set-output name=plugins::$PLUGINS"
+
+  # Bumpt the plugins on RPM
+  bump_plugin_rpm:
+    needs: plugin_list
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        plugin:
+          - ${{ fromJson(needs.plugin_list.outputs.plugins) }}
+    env:
+      PLUGIN_DIR: "packages/plugins/rubygem-${{ matrix.plugin }}"
+      SPEC_FILE: "packages/plugins/rubygem-${{ matrix.plugin }}/rubygem-${{ matrix.plugin }}.spec"
+    steps:
+    - name: Install dependencies
+      run: |
+        sudo apt-get install -y rpm curl jq git-annex python3-pip
+        sudo gem install gem2rpm
+        sudo apt-get install -y libcurl4-openssl-dev
+        sudo gem install gem-compare
+        sudo pip3 install semver
+        sudo curl --create-dirs -o /usr/local/bin/spectool https://pagure.io/rpmdevtools/raw/26a8abc746fba9c0b32eb899b96c92841a37855a/f/spectool.in
+        sudo chmod +x /usr/local/bin/spectool
+    - name: Fetch latest plugin gem info
+      id: latest_gem
+      run: |
+        curl_output=$(curl -s https://rubygems.org/api/v1/gems/${GEM_NAME}.json)
+        echo "::set-output name=version::$(jq -r .version <<<$curl_output)"
+        echo "::set-output name=uri::$(jq -r .gem_uri <<<$curl_output)"
+      env:
+        GEM_NAME: ${{ matrix.plugin }}
+    - name: Checkout RPM
+      uses: actions/checkout@v2
+      with:
+        ref: rpm/develop
+    - name: Initialize git annex
+      run: git annex init
+    - name: Try to bump plugin
+      run: ./bump_rpm.sh $PLUGIN_DIR -a "$CHANGE_AUTHOR" --no-commit
+      env:
+        CHANGE_AUTHOR: Github Actions <noreply@github.com>
+    - name: Open a PR
+      uses: peter-evans/create-pull-request@v3
+      with:
+        commit-message: "Updates ${{ matrix.plugin }} to ${{ steps.latest_gem.outputs.version }}"
```

I'd like to avoid duplicating the logic from `bump_rpm.sh` but if you do, please copy its syntax. It uses the convention as specified in https://cbea.ms/git-commit/#imperative.
```suggestion
        commit-message: "Update ${{ matrix.plugin }} to ${{ steps.latest_gem.outputs.version }}"
```